### PR TITLE
[Wfw-8695ap6m2] display chat for a selected friend

### PR DESCRIPTION
**Pull request by Svitlana-G**
https://app.clickup.com/t/8695ap6m2

### CODE REVIEW

File: src/types/Chat.ts
Comment (oce4ka) [inline src/types/Chat.ts:1]: Why not to use here Date type? Is string better in this case?

File: src/pages/MessagesPage.tsx [hunk 1]
```diff
@@ -12,10 +12,15 @@ import theme from './../styles/createTheme'
...
 import StartChatting from 'components/chat/StartChatting'
+import DisplayingChat from 'components/chat/DisplayingChat'
+import messages from '../components/chat/chat.json'
 
 const MessagesPage = () => {
   const { classes } = useStyles()
   const [selectedChat, setSelectedChat] = useState<UserChatProfile | null>(null)
+  const userId = '1'
+
+  const frienId = messages.participants.find((el) => el !== userId)
```
Comment (oce4ka) [inline src/pages/MessagesPage.tsx:23]: Typo or on purpose? frienId ?= friendId

File: src/types/Chat.ts
Comment (oce4ka) [reply src/types/Chat.ts:1]: Upd. Reviver function may add more complicity

**Review by oce4ka — APPROVED** [review]
I think everything is good. There might be a typo, but nothing critical.

File: src/types/Chat.ts
Comment (Svitlana-G) [reply src/types/Chat.ts:1]: String is human-readable
In the second comment, I didn't understand what was meant (about reviver function)

**Pull request merged**